README.md

```diff
@@ -2,11 +2,17 @@
 
 [![Build Status](https://travis-ci.org/jts/nanopolish.svg?branch=master)](https://travis-ci.org/jts/nanopolish)
 
-Software package for signal-level analysis of Oxford Nanopore sequencing data.
+Software package for signal-level analysis of Oxford Nanopore sequencing data. Nanopolish is designed to calculate a new consensus sequence for a draft genome assembly produced by a program like [CANU](https://github.com/marbl/canu). It does not error correct reads.
 
 ## Dependencies
 
-The program requires [libhdf5](http://www.hdfgroup.org/HDF5/release/obtain5.html) and a compiler that supports C++11. Development of the code is performed using [gcc-4.8](https://gcc.gnu.org/gcc-4.8/). libhdf5 can be automatically installed by the Makefile if you do not have it already (see below).
+[libhdf5](http://www.hdfgroup.org/HDF5/release/obtain5.html). It is automatically downloaded and compiled during `make` step but you can disable it with: `HDF5=nofetch make`. It is not necessary to install it (and `make install` is not called either). The nanopolish binary will link using a libhdf5.a (statically).
```

Comment:
I am lazy to check now but actually didn't I say it overwrite my site-wide installation? Well, you will probably change this Makefile code anyway sooner or later.